microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -10,35 +10,34 @@ keywords: microsoft edge, web development, f12 tools, devtools
 ---
```

Comment:
description: Force Microsoft Edge DevTools into Color Scheme Preview mode.
That description implies that the article is about changing the theme of the DevTools portion of the window:
Is the article actually about changing the rendered webpage portion of the window instead?

The tab text in the browser shows "Force Microsoft Edge DevTools", which reads oddly.  The tab is probably populated by the "title:" field.  A useful string for one of the YAML fields might be "Render the webpage with light or dark theme emulation".

Update the "date:".

Make the "title:" field match the # (top-of-page) title.  Currently those are:
title: Force Microsoft Edge DevTools into Color Scheme Preview mode (CSS Prefers Color Scheme)
# Dark or light color scheme simulation 
If you change the # title, check the TOC.yml too and do full-text search in repo for the present filename, to update link text in any articles that link to the present article.

Reply:
I would personally prefer to keep the title shorter and drop the "in the rendered page" part. My reasoning for this is that all other pages in the Rendering section are about changing things in the page (as opposed to devtools). So I don't think there is too much of a risk for readers to get confused as to where the color scheme will apply, especially if the content of the article makes it clear.